web/app/scripts/src/services/api.es.js

```diff
@@ -17,10 +22,16 @@ export const api = function ($http, PROJECT_SETTINGS) {
     }
 };
 
-export const service = [
-    '$http',
-    'PROJECT_SETTINGS',
-    api
-];
+export const module = {
+    moduleName: 'incuna.surveys-api',
+    componentName: 'API'
+};
+
+angular.module(module.moduleName, [ProjectConfig.moduleName])
+    .service([
+        '$http',
+        ProjectConfig.componentName,
+        API
```

Comment:
>I expose it for easier mocking

👍 

>I think it's a bit easier to just pass in a mock module than to use the `injector()` syntax.

Not sure what you mean by that. Could you explain in examples?

> 2) you still see the modules being imported where the function is defined

I mean the injector dependencies, `'$http'` and `ProjectConfig.componentName`. I think it's important that they're listed right next to the function definition that accepts them as parameters in order.
E.g.
```js
module.service('Foo', [
    'dep1',
    'dep2,
    function (dep1, dep2) {}
]);
```
Instead of
```js
var FooService = function (dep1, dep2) {};
// ...
// ...
// ...
module.service('Foo', [
    'dep1',
    'dep2,
    FooService
]);
```